paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -99,15 +97,16 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     val saveForFutureUse: StateFlow<Boolean> = saveForFutureUseElement.controller.saveForFutureUse
         .stateIn(viewModelScope, SharingStarted.Lazily, args.formArgs.showCheckbox)
 
-    val requiredFields = name
-        .map { it.isNotBlank() }
-        .combine(
-            emailElement.getFormFieldValueFlow().map { formFieldsList ->
-                formFieldsList.firstOrNull()?.second?.isComplete ?: false
-            }
-        ) { validName, validEmail ->
-            validName && validEmail
+    val requiredFields = combine(
```

Comment:
nit: requiredFields made me think it would be the actual required fields, but I think this is just if the requiredFields are present, so maybe requiredFieldPresent would make it more readable where used.